jerry-core/parser/js/js-parser.c

```diff
@@ -2898,14 +2904,14 @@ parser_parse_script (const uint8_t *arg_list_p, /**< function argument list */
       /* It is unlikely that memory can be allocated in an out-of-memory
        * situation. However, a simple value can still be thrown. */
       jcontext_raise_exception (ECMA_VALUE_NULL);
-      return ECMA_VALUE_ERROR;
+      return NULL;
     }
 
     if (parser_error.error == PARSER_ERR_INVALID_REGEXP)
```

Comment:
This `if` and the `if` above sems to have the same body. Can they be merged?

Reply:
They seem distinct to me. In the first case were raising a new exception, while in the latter the context already contains one. I guess the return statement could be factored out though.

Reply:
It was my mistake. Leave them as they are now.